MathlibTest/Perm.lean

```diff
@@ -0,0 +1,12 @@
+
+import Mathlib.GroupTheory.Perm.Cycle.Concrete
+import Mathlib.Tactic.DeriveFintype
+
+open Equiv
+
+#guard with_decl_name% ex₁ unsafe (reprStr (1 : Perm (Fin 4))) == "1"
+#guard with_decl_name% ex₂ unsafe (reprStr (c[0, 1] : Perm (Fin 4))) == "c[0, 1]"
+#guard with_decl_name% ex₃
+  unsafe (reprStr (c[0, 1] * c[2, 3] : Perm (Fin 4))) == "c[0, 1] * c[2, 3]"
+#guard with_decl_name% ex₄
+  unsafe (reprPrec (c[0, 1] * c[2, 3] : Perm (Fin 4)) 70).pretty == "(c[0, 1] * c[2, 3])"
```

Comment:
What does your `Repr` too on permutations that are not given in disjoint cycles form?
```suggestion
  unsafe (reprPrec (c[0, 1] * c[2, 3] : Perm (Fin 4)) 70).pretty == "(c[0, 1] * c[2, 3])"
#guard with_decl_name% ex₅
  unsafe (reprPrec (c[0, 1] * c[1, 2] : Perm (Fin 3)) 70).pretty == "???"
```